target/linux/lantiq/files-4.14/arch/mips/boot/dts/FRITZ7362SL.dts

```diff
@@ -0,0 +1,286 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "vr9.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mips/lantiq_rcu_gphy.h>
+
+/ {
+	compatible = "avm,fritz7362sl", "lantiq,xway", "lantiq,vr9";
+	model = "AVM FRITZ!Box 7362 SL";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200";
+	};
+
+	aliases {
+		led-boot = &power_green;
+		led-failsafe = &power_red;
+		led-running = &power_green;
+		led-upgrade = &power_green;
+
+		led-dsl = &info_green;
+		led-wifi = &wifi;
+	};
+
+	memory@0 {
+		reg = <0x0 0x8000000>;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
```

Comment:
@danijelt You can drop `#address-cells = <1>;` and `#size-cells = <0>;` as it's not necessary.

Reply:
Done.